performance_regression_row_level_repair_test.py

```diff
@@ -211,6 +211,39 @@ def _pre_fill_schema(table_name=self.TABLE_NAME, partitions=self.PARTITIONS, row
                                                                        )
                 session.execute(stmt)
 
+        # Util functions ===============================================================================================
+
```

Comment:
I don't understand what I see here...

Reply:
this code is used to fill various types of large partitions. it may be replaced by scylla-bench stress and will become obsolete.